etc/authordb.yaml

```
@@ -3774,9 +3787,9 @@ authors:
     orcid: null
   chandlerc:
     affil:
-    - NAU
-    altaffil: []
-    email: null
+    - LINCC
+    altaffil: [Washington, NAU]
+    email: coc123@uw.edu
```

```suggestion
    email: coc123@Washington
```
(so you don't have your email fully obvious).

---

Done.